static/scripts/view.js

```diff
@@ -137,17 +140,17 @@ function editEnclosure(button) {
 
   showEnclosure(row);
 
-  // Text box
-  var textBox = document.createElement('textarea');
-  textBox.setAttribute('name', 'noteContent');
-  textBox.setAttribute('rows', 20);
+  // Mark the textareas as being edited
```

Comment:
I'd also mention the larger comment  in the .py file here which explains the edit and send textboxes. 

Perhaps also mention here what the "pre" box is for.